examples/slurm/container-entrypoint.sh

```diff
@@ -16,6 +16,10 @@
 
 # Start the scheduler on the rank 0 node
 if [[ -z "$SLURM_NODEID" ]] || [[ $SLURM_NODEID == 0 ]]; then
+  # Make the directories needed
+  mkdir -p $LOGDIR
+  mkdir -p $PROFILESDIR
```

Comment:
This is run inside the container ?

Reply:
Yes you're correct. It's a good call out, and it makes me think we maybe should've done this from the beginning since the contents of `$LOGDIR` and `$PROFILESDIR` get written inside the container, so they ought to be initialized in it too. Let me know if you disagree.

Reply:
So, i think because logs are accessed from outside the container so we should make it clear where the path is from outside the container,  I think to make it clear we should echo `$LOGDIR` and  `$PROFILESDIR`  to help someone debugging this.

Reply:
Added echo and comments.